linkml_model/model/schema/meta.yaml

```diff
@@ -2541,6 +2536,145 @@ slots:
+    domain: schema_definition
+    range: date
+    inherited: true
+    slot_uri: dcterms:modified
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:modified
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/modified
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  rights:
+    description: Property rights of the vocabulary.
+    domain: schema_definition
+    range: string
+    inherited: true
+    slot_uri: dcterms:rights
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:rights
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/rights
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  licence:
+    description: >-
+      Licence of the vocabulary. 
+      Recommended practice is to refer to a rights statement with a URI. 
+      If this is not possible or feasible, a literal value (name, label, or short text) may be provided.
+    domain: schema_definition
+    range: string
+    inherited: true
+    slot_uri: cc:licence
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - cc:licence
+    see_also:
+      - https://creativecommons.org/ns
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  creator:
+    description: >-
+      Creator(s) of the vocabulary.
+      Recommended practice is to identify the creator with a URI. 
+      If this is not possible or feasible, a literal value that identifies the creator may be provided.
+    domain: schema_definition
+    range: string
+    inherited: true
+    slot_uri: dcterms:creator
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dcterms:creator
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/creator
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  contributor:
+    description: >-
+      Contributor(s) of the vocabulary.
+      Recommended practice is to identify the contributor with a URI. 
+      If this is not possible or feasible, a literal value that identifies the contributor may be provided.
+    domain: schema_definition
+    range: string
+    inherited: true
+    slot_uri: dc:contributor
+    in_subset:
+      - OwlProfile
+    exact_mappings:
+      - dc:contributor
+    see_also:
+      - https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/contributor
+      - https://lov.linkeddata.es/Recommendations_Vocabulary_Design.pdf
+
+  publisher:
+    description: >-
+      Publisher(s) of the vocabulary.
+      Recommended practice is to identify the contributor with a URI. 
+      If this is not possible or feasible, a literal value that identifies the publisher may be provided.
+    domain: schema_definition
+    range: string
```

Comment:
not so keen on this, but I guess I will lose this battle.. (why not uriorcurie)?